common-functions.js

```diff
@@ -0,0 +1,125 @@
+var dateArray =[];
+var dateArray2 =[];
+
+function dateFormatSlashDDMM(a) {
+	this.dateArray=a;
+	var output1 = this.dateArray[2]+"/"+this.dateArray[1]+"/"+this.dateArray[0];
+	document.getElementById("op1").innerHTML = output1;
```

Comment:
return the output value from here and do the document.getElement part from main.js
e.g : 
return output;

var result = COMMON_FUNCTION.dateFormatSlashDDMM(date);
document.getElementById("op1").innerHTML = result ;